dev/test/commands/generate_release_command_test.rb

```diff
@@ -137,8 +137,68 @@ class GenerateReleaseCommandTest < TestCase
       to_shopify_content = File.read(File.expand_path("data/integrations/shopify/mappings/to_shopify.yml", @tmp_base_path))
       from_shopify_content = File.read(File.expand_path("data/integrations/shopify/mappings/from_shopify.yml", @tmp_base_path))
 
-      assert_equal "output_version: #{@version}", to_shopify_content
-      assert_equal "input_version: #{@version}", from_shopify_content
+      assert_equal "output_taxonomy: shopify/#{@version}", to_shopify_content
+      assert_equal "input_taxonomy: shopify/#{@next_version}", from_shopify_content
+    end
+
+    test "execute updates taxonomy fields in mapping files" do
+      command = GenerateReleaseCommand.new(current_version: @version, next_version: @next_version)
+      command.execute
+
+      # After execute, to_shopify should have the stable version
+      to_shopify_content = File.read(File.expand_path("data/integrations/shopify/mappings/to_shopify.yml", @tmp_base_path))
+      assert_match "output_taxonomy: shopify/#{@version}", to_shopify_content
+      refute_match "output_taxonomy: shopify/2023-12-unstable", to_shopify_content
+
+      # After execute, from_shopify should have the next unstable version
+      from_shopify_content = File.read(File.expand_path("data/integrations/shopify/mappings/from_shopify.yml", @tmp_base_path))
+      assert_match "input_taxonomy: shopify/#{@next_version}", from_shopify_content
+      refute_match "input_taxonomy: shopify/2023-12-unstable", from_shopify_content
+    end
+
+    test "mapping files transformations pipeline during execute stages" do
```

Comment:
This test name seems pretty opaque, how about:

```suggestion
    test "execute updates input and output taxonomies for integrations in two stages" do
```